internal/lotus/blocks.go

```diff
@@ -0,0 +1,76 @@
+package lotus
+
+import (
+	"github.com/NodeFactoryIo/hactar-daemon/internal/lotus/requests/lotus"
+	log "github.com/sirupsen/logrus"
+)
+
+type BlocksService interface {
+	GetLastBlock() (*LastBlockResponse, error)
+}
+
+type blocksService struct {
+	client *Client
+}
+
+type LastBlockResponse struct {
+	Cids []struct {
+		Root string `json:"/"`
+	} `json:"Cids"`
+	Blocks []struct {
```

Comment:
is this structure necessary for parsing since we don't use all values? In case they change some things in the API, we'll have a hard time to track it down and update.